cosmic-settings/src/pages/system/mod.rs

```diff
@@ -4,6 +4,8 @@
 #[cfg(feature = "page-about")]
 pub mod about;
 
+#[cfg(feature = "page-about")]
```

Comment:
I resolved the conflict with https://github.com/pop-os/cosmic-settings/pull/1597, but the spacing seems weird here. Do we want blank lines between each of these `#[cfg(feature = "page-about")]` blocks, or not? It seems like rustfmt isn't doing anything about it.